spectral_fitting/fitting_with_spectralcube.ipynb

```diff
@@ -0,0 +1,697 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python_defaultSpec_1599602071388",
+   "display_name": "Python 3.6.9 64-bit ('anaconda3': virtualenv)"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fitting with spectral-cube and astropy #\n",
+    "\n",
+    "## Authors ##\n",
+    "Eric Koch, (add here)\n",
+    "\n",
+    "## Learning Goals ##\n",
+    "\n",
+    "* Fitting 1D models to spectra in a spectral-line data cube\n",
+    "* Fitting 2D models to spectral channels or 2D projections\n",
+    "* Enabling dask mode for large data cubes\n",
+    "\n",
+    "## Keywords ##\n",
+    "\n",
+    "radio astronomy, spectral-line data cubes, spectral fitting, spatial fitting\n",
+    "\n",
+    "## Summary ##\n",
+    "\n",
+    "This tutorial demonstrates methods to fit models to a spectral-cube object using [astropy.modeling](https://docs.astropy.org/en/stable/modeling/).\n",
+    "\n",
+    "## Requires ##\n",
+    "\n",
+    "```\n",
+    "pip install astropy\n",
+    "pip install spectral-cube\n",
+    "pip install radio-beam\n",
+    "pip install dask\n",
+    "pip install aplpy\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from spectral_cube import SpectralCube, Projection\n",
+    "import astropy.units as u\n",
+    "from astropy.io import fits\n",
+    "from astropy.modeling import models, fitting\n",
+    "from astropy.convolution import convolve_models\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Fitting models to both spatial maps and spectra are amongst the most common operations involving spectral-line data cubes.\n",
+    "\n",
+    "In this tutorial, we explore how to fit models to a `SpectralCube` using `astropy.modeling`. To show this, we will use an ALMA mosaic observed with the ACA, or 7-m array. This mosaic is one component of an ALMA project to map CO(2-1) in M33 (Project ID 2019.1.01182.S).\n",
```

Comment:
Could link the project ID to an online abstract/program description if such a thing exists?

Reply:
@keflavich - do you know if you can link to a persistent search URL on the ALMA archive? It wasn't clear to me on the new archive site

Reply:
Good question.  Have to ask Adrian Damian - I don't know!

Reply:
I think we can just stick with the project ID for now. I've made an issue to come back once the full data products are hosted somewhere (#14 )

Reply:
I've added a link to the Zenodo tutorial data for now, along with the ALMA project ID.